ui/src/pages/Admin/AllStories/index.tsx

```diff
@@ -533,8 +552,8 @@ export const AllStories: React.FC = () => {
 
   const toast = React.useRef(null);
 
-  const showToast = (message: string) => {
-    toast.current.showToast(message);
+  const showToast = (message: string, icon: any) => {
```

Comment:
can we address this warning for the type for icon?

Reply:
wait what warning?

Reply:
![image](https://user-images.githubusercontent.com/22552967/116946230-96f18c00-ac47-11eb-9d54-3886919a05ad.png)